blog/battle-of-blockchains.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "The Battle of Blockchains: PoW vs PoS Unveiled"
+date: "17/04/2024"
+desc: "A perspective on the long-standing battle between proof-of-work and proof-of-stake blockchains."
+authors: "zeta"
+draft: true
+---
+
+Since the dawn of civilization, we have managed to escape many physical
+barriers of nature. With enough money, a human construct devoid of any
+physical worth, one can buy the essentials to cover all their basic
+needs. Yet our bodies exemplify the intangible nature of money: for
+instance, the fruits of physical exercise, such as a healthier and
+longer lifespan on average, can only be reaped through effort and cannot
+be bought. Nonetheless, a debate arises when considering the advantages
+that financial resources can bring, such as access to health supplements
+or advanced exercise equipment, thus posing the question of their true
+worth. *Is the essence of effort so easily substituted by financial
+resources?*
+
+The quest to answer this question is pivotal for the fabric of modern
+society as well as the realm of blockchains. Indeed, it is a common
+belief that blockchains operate optimally when users with more money
+imply their wealth to secure blockchains. We contest this folklore
+wisdom, advocating for blockchains operated via effort or ,in machine
+language, computational power; the so-called Proof-of-Work (PoW). Put
+simply, we argue that blockchains, much like our bodies, are more secure
+with "work".
+
+Delving into the foundations of blockchains, two properties prevail as
+their backbone [@bitcoinbackboneprotocol]: safety, ensuring that
```

Comment:
```suggestion
the backbone [@bitcoinbackboneprotocol] of the blockchain protocols: safety, ensuring that
```

Reply:
I think the current wording makes sense.